scale-test/fioload-template.yaml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: @podname@
+  labels:
+    app: @testid@
+spec:
+  restartPolicy: Never
+  @nodeSelector@:
+    type: worker
+  containers:
+  - command:
+    - sh
+    - -c
+    - 'fio --blocksize=@bs@ --verify=0 --randrepeat=0 --filename=/mnt/file1 --ioengine=libaio --readwrite=randread --runtime=60s --time_based --ramp_time=2s --size=3G --name=test --numjobs=8 --direct=1 --iodepth=2 --group_reporting > /tmp/fiotest; exit 0;'
+    image: ljishen/fio
+    name: fioload
+    volumeMounts:
+    - mountPath: /mnt
+      name: data-volume
+  - command: ["/bin/sh", "-c"]
+    args:
+      - cnt=0;
+        while [ $cnt -ne 1 ];
+        do
+          sleep 5;
+          cnt=`ls -ltr /mnt/file1 2>/dev/null | wc -l`;
+        done;
+        sleep 5;
+        fio --blocksize=@bs@ --verify=0 --randrepeat=0 --filename=/mnt/file2 --ioengine=libaio --readwrite=randread --runtime=15s --time_based --ramp_time=2s --size=3G --name=test --direct=1 --group_reporting > /fio-results/@output@;
```

Comment:
You are running fio twice? Can you explain what the idea behind that is? Also, why do you need to sleep for the file to be created? If you want to have jobs completed, or started in a certain order, the fio configuration files let you do that. 

Also, in both cases the verify option has no effect, you can remove it. Lastly the options used in the first test are not the same as in the last test.